doc/whats_new/v1.0.rst

```diff
@@ -95,12 +95,30 @@ Changelog
   in multicore settings. :pr:`19052` by
   :user:`Yusuke Nagasaka <YusukeNagasaka>`.
 
-- |API| :class:`cluster.Birch` attributes, `fit_` and `partial_fit_`, are
-  deprecated and will be removed in 1.2. :pr:`19297` by `Thomas Fan`_.
+- |Efficiency| :class:`cluster.MiniBatchKMeans` is now faster in multicore
```

Comment:
It seems that these entries are due to some issue while merging `main` into your branch?

Reply:
Ah yes, I will fix this later.